container-registry/container-analysis/src/main/java/com/example/containeranalysis/Samples.java

```diff
@@ -43,307 +52,288 @@
-
-      client.deleteNote(noteName);
-    }
+  public static void deleteNote(GrafeasV1Beta1Client client, String noteId, String projectId) {
+    final String noteName = client.formatNoteName(projectId, noteId);
+    client.deleteNote(noteName);
   }
   // [END delete_note]
 
   // [START delete_occurrence]
   /**
-   * Deletes an existing occurrence
-   *
+   * Deletes an existing Occurrence from the server
+   * @param client The Grafeas client used to perform the API requests.
    * @param occurrenceName the name of the occurrence to delete
    *                       format: "projects/{projectId}/occurrences/{occurrence_id}"
-   * @throws Exception on errors while closing the client
    */
-  public static void deleteOccurrence(String occurrenceName) throws Exception {
-    try (ContainerAnalysisClient client = ContainerAnalysisClient.create()) {
-      client.deleteOccurrence(occurrenceName);
-    }
+  public static void deleteOccurrence(GrafeasV1Beta1Client client, String occurrenceName) {
+    client.deleteOccurrence(occurrenceName);
   }
   // [END delete_occurrence]
 
 
   // [START get_occurrence]
   /**
-   * Retrieves an occurrence based on its name
-   *
+   * Retrieves a specified Occurrence from the server
+   * @param client The Grafeas client used to perform the API requests.
    * @param occurrenceName the name of the occurrence to delete
    *                       format: "projects/{projectId}/occurrences/{occurrence_id}"
-   * @return the specified Occurrence object
-   * @throws Exception on errors while closing the client
+   * @return the requested Occurrence object
    */
-  public static Occurrence getOccurrence(String occurrenceName) throws  Exception {
-    try (ContainerAnalysisClient client = ContainerAnalysisClient.create()) {
-      return client.getOccurrence(occurrenceName);
-    }
+  public static Occurrence getOccurrence(GrafeasV1Beta1Client client, String occurrenceName) {
+    return client.getOccurrence(occurrenceName);
   }
   // [END get_occurrence]
 
   // [START get_note]
   /**
-   * Retrieves a note based on its noteId and projectId
-   *
+   * Retrieves a specified Note from the server
+   * @param client The Grafeas client used to perform the API requests.
    * @param noteId the note's unique identifier
-   * @param projectId the project's unique identifier
-   * @return the specified Note object
-   * @throws Exception on errors while closing the client
+   * @param projectId the GCP project the Note belongs to
+   * @return the requested Note object
    */
-  public static Note getNote(String noteId, String projectId) throws  Exception {
-    try (ContainerAnalysisClient client = ContainerAnalysisClient.create()) {
-      final String noteName = client.formatNoteName(projectId, noteId);
+  public static Note getNote(GrafeasV1Beta1Client client, String noteId, String projectId) {
+    final String noteName = client.formatNoteName(projectId, noteId);
```

Comment:
FYI the Java lib has been updated to now have the methods with the resource name types:
https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-clients/google-cloud-containeranalysis/src/main/java/com/google/cloud/devtools/containeranalysis/v1beta1

E.g., instead of using the formatter to create a string, you can do:
NoteName name = NoteName.of("[PROJECT]", "[NOTE]");

to use the NoteName type.

Reply:
Same for all other resource types.